css/css-scoping/slotted-details-content-ref.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<style>
+  ::details-content { color: green }
```

Comment:
I would make this `body { color: green}` so that the test fails when `::details-content` isn't supported.

Reply:
body would make the summary green too, I can wrap the text inside the details into a green span.

Reply:
Sounds good.  (I forgot about the autogenerated summary text.)